sql/src/main/java/com/rilldata/calcite/DependencyFinder.java

```diff
@@ -0,0 +1,53 @@
+package com.rilldata.calcite;
+
+import org.apache.calcite.sql.SqlBasicCall;
+import org.apache.calcite.sql.SqlCall;
+import org.apache.calcite.sql.SqlIdentifier;
+import org.apache.calcite.sql.SqlJoin;
+import org.apache.calcite.sql.SqlNode;
+import org.apache.calcite.sql.SqlSelect;
+import org.apache.calcite.sql.ddl.SqlCreateTable;
+import org.apache.calcite.sql.ddl.SqlCreateView;
+import org.apache.calcite.sql.fun.SqlStdOperatorTable;
+import org.apache.calcite.sql.util.SqlBasicVisitor;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DependencyFinder extends SqlBasicVisitor<List<String>>
```

Comment:
Can you write brief description of what this class does ?

Reply:
Added